Detectors/TPC/simulation/include/TPCSimulation/DigitCRU.h

```diff
@@ -41,6 +41,10 @@ class DigitCRU{
     /// \return Size of the time bin container
     size_t getSize() const {return mTimeBins.size();}
 
+    /// Get the container
+    /// \return container
+    const std::deque<std::unique_ptr<DigitTime>>& getContainer() const { return mTimeBins; }
+
```

Comment:
getContainer sounds a bit too generic. Can we consider having a more concrete name? (getTimeBins or getTimeBinContainer ...)